sql/sql_explain.cc

```diff
@@ -351,9 +351,9 @@ int print_explain_row(select_result_sink *result,
                       mem_root);
   
   /* 'possible_keys' */
+  StringBuffer<64> possible_keys_buf;
   if (possible_keys && !possible_keys->is_empty())
   {
-    StringBuffer<64> possible_keys_buf;
```

Comment:
This fix is correct, it will be merged. I don't really agree with how the items are used in this context on thd mem_root, but it's perfectly fine to fix the memory access violation like this.